pool/pool.c

```diff
@@ -588,28 +588,30 @@ m0_pool_version_find(struct m0_pools_common *pc, const struct m0_fid *id)
 	struct m0_pool_version *pv;
 	struct m0_conf_root    *root;
 	struct m0_conf_pver    *pver = NULL;
-	int                     rc;
+	int                     rc = 0;
 
 	M0_ENTRY(FID_F, FID_P(id));
 	M0_PRE(pc != NULL);
 
 	m0_mutex_lock(&pc->pc_mutex);
 	pv = m0_pool_version_lookup(pc, id);
-	if (pv != NULL) {
-		rc = 0;
+	if (pv != NULL)
 		goto end;
-	}
+
 	rc = m0_confc_root_open(pc->pc_confc, &root);
 	if (rc != 0) {
-		M0_LOG(M0_ERROR, "Cannot open root object");
+		M0_LOG(M0_ERROR, "Failed to open conf-root object: rc=%d", rc);
 		goto end;
 	}
 	rc = m0_conf_pver_find_by_fid(id, root, &pver) ?:
 		m0_pool_version_append(pc, pver, &pv);
 	m0_confc_close(&root->rt_obj);
-end:
+ end:
```

Comment:
Why is this leading space added?

Reply:
To not mix function names (which can start in the beginning of the line sometimes) with the goto labels. Don't remember already where I picked it from, but I liked the idea.